<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExQHostInfo</class>
 <widget class="QWidget" name="ExQHostInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExQHostInfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>www.baidu.com</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnFindIP">
       <property name="text">
        <string>QHostInfo查询左侧域名IP地址</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnAllInterfaces">
       <property name="text">
        <string>QNetworkInterface::allInterfaces()</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnGetHostInfo">
       <property name="text">
        <string>QHostInfo查询主机名和IP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnAllAddresses">
       <property name="text">
        <string>QNetworkInterface::allAddresses()</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>只显示IPv4协议的网络信息</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btnClean">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
